### ISSUE TITLE
Convert to Hubitat Generic Component Child Devices

### ISSUE THREAD

**Issue by apwelsh** [InovelliUSA/Hubitat#36]
Any plans to convert from custom child devices to the Hubitat Generic Component Child Devices?    Doing so reduces the number of custom device drivers to install on the HE.  If willing, I am happy to begin the process for various devices one-at-a-time.

**Comment by InovelliUSA** [comment]
We actually have already done this for the Gen 2 devices. Are you talking about the Gen 1 devices?

**Comment by apwelsh** [comment]
I guess it was the NZW30 and NZW31 (both w/scene since the firmware update).  I did not realize that this was done for Gen 2 but not Gen 1 devices.

**Comment by InovelliUSA** [comment]
Yeah, it has been done for Gen 2 devices. You'll notice that I left the old child methods in the parent driver so that it will work with both the Inovelli child devices and the Hubitat ones. That way, if someone updates the parent, things will still work for them.
